Source/WTF/wtf/ASCIICType.h

```diff
@@ -49,7 +49,9 @@ template<typename CharacterType> constexpr bool isASCIIHexDigit(CharacterType);
 template<typename CharacterType> constexpr bool isASCIILower(CharacterType);
 template<typename CharacterType> constexpr bool isASCIIOctalDigit(CharacterType);
 template<typename CharacterType> constexpr bool isASCIIPrintable(CharacterType);
-template<typename CharacterType> constexpr bool isASCIISpace(CharacterType);
+template<typename CharacterType> constexpr bool isASCIIWhitespace(CharacterType);
+template<typename CharacterType> constexpr bool isNotASCIIWhitespace(CharacterType);
+template<typename CharacterType> constexpr bool isCCompatibleASCIIWhitespace(CharacterType);
```

Comment:
I don’t think "C compatible" is the ideal phrase here. The broader version of whitespace that includes U+000B is:

1) Consistent with Unicode’s whitespace definition.
2) Consistent with the "C" locale behavior of the C standard library `isspace` function.

The second doesn’t really say "C compatible" to me. I would suggest a name more like `isTraditionalASCIIWhitespace` or `isOldFashionedASCIIWhitespace` or `isUnicodeCompatibleASCIIWhitespace`. I don’t love any of those, but I think they are all clearer than "C compatible".

Reply:
I went with `isUnicodeCompatibleASCIIWhitespace` for now. Although for 8-bit strings we probably need a Latin1-compatible definition for it to be useful.

Reply:
We have that elsewhere for Latin-1. I'll find it for you.